__tests__/app.test.js

```diff
@@ -121,4 +122,56 @@ describe('/api/reviews/:review_id', () => {
       })
     });
   });
-});
\ No newline at end of file
+});
+
+describe('/api/reviews/:review_id/comments', () => {
+  describe('get request', () => {
+    test('get all comments of the review with the given id', () => {
+      return request(app)
+      .get('/api/reviews/3/comments')
+      .expect(200)
+      .then(({body})=>{
+        expect(body.comments).toBeInstanceOf(Array)
+        expect(body.comments).toHaveLength(3)
+
+        body.comments.forEach((comment)=>{
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: expect.any(Number)
+          })
+        })
+      })
+    });
+
+    test('should give error if id given is out of range', () => {
+      return request(app)
+      .get('/api/reviews/100/comments')
+      .expect(404)
+      .then(({body}) => {
+        expect(body.msg).toEqual('ID does not exist')
+      })
+    });
+
+    test('should give error if id given is out of range', () => {
+      return request(app)
+      .get('/api/reviews/hello/comments')
+      .expect(400)
+      .then(({body}) => {
+        expect(body.msg).toEqual('Invalid parameter')
+      })
+    });
+
+    test('should give error if id given is out of range', () => {
+      return request(app)
+      .get('/api/reviews/1/comments')
+      .expect(404)
+      .then(({body}) => {
+        expect(body.msg).toEqual('No comments in this review')
+      })
```

Comment:
The above 3 tests all have the same description - please change them to be more reflective of what you are specifically asserting in each one.
Also - this is a design decision - but if the review DOES exist, just with no comments, we would generally expect a 200 with an empty array of comments.
You can think of this reasoning from more of a front end perspective